website/src/components/canvas/Sidebar.jsx

```diff
@@ -61,8 +61,9 @@ class Sidebar extends React.Component {
    * @returns {void}
    */
   addElement() {
-    /* TODO: Make this take a specific module*/
-    FBHelper.addElement(this.props.currentCanvas, 'abcd',
+    /* NOTE: leaving this here for ease of testing */
+    /* TODO: remove before deploying */
```

Comment:
Only when we are done -- I'm leaving the add element button in until you guys can finish debugging some of the current bugs.